airbyte-integrations/bases/base-normalization/normalization/transform_catalog/stream_processor.py

```diff
@@ -203,23 +207,39 @@ def process(self) -> List["StreamProcessor"]:
             return []
 
         column_names = self.extract_column_names()
+        column_count = len(column_names)
```

Comment:
thought for subsequent: 
* this is getting kinda big - we should break this up if we are going to add another change to this.
* we are getting to a size where I feel like we should start running MyPy.